js/base/Exchange.js

```diff
@@ -778,16 +778,6 @@ module.exports = class Exchange {
         throw new NotSupported (this.id + ' fetchTickers not supported yet')
     }
 
-    purgeCachedOrders (before) {
```

Comment:
It's in our docs, we can't delete it until we remove the orders cache from CCXT entirely.

Reply:
ok nvm i see yobit and exmo still use it